pages/agent/v3/upgrading.md.erb

```diff
@@ -7,13 +7,13 @@ The Buildkite Agent has changed a lot in v3, but we've tried to keep it as backw
 ## What's changed
 
 * [Plugins](/docs/agent/v3/plugins) for easily sharing functionality between pipelines and customizing how agents behave
-* The bootstrap (run as a sub-process for every job) has moved from a [shell script](https://github.com/buildkite/agent/blob/2-6-stable/templates/bootstrap.sh) to a sub-command of the `buildkite-agent bootstrap`. This means it's written in golang and cross-platform.
+* The bootstrap (run as a sub-process for every job) has moved from a [shell script](https://github.com/buildkite/agent/blob/2-6-stable/templates/bootstrap.sh) to [`buildkite-agent bootstrap`](cli_bootstrap). This means it's written in golang and cross-platform.
 * Agent meta-data has been renamed to "tags"
 * Built-in [Docker and Docker Compose support](/docs/builds/docker-containerized-builds) has been deprecated and moved to dedicated plugins, [docker-compose](https://github.com/buildkite-plugins/docker-compose-buildkite-plugin) and [docker](https://github.com/buildkite-plugins/docker-buildkite-plugin).
-* Variable interpolation in pipeline.yml
+* [Variable interpolation in pipeline.yml](cli_pipeline)
 * Much better Windows support, including .BAT hooks support
-* Build annotations
-* Default flags for `git clean` include `-x`
```

Comment:
Is this no longer something that's changed?

Reply:
Oh I see the bit later on, nevermind 😅

Reply:
Yeah, I thought it was clearer to explain the intention vs just the technical change. Is it clear?

Reply:
Yeah the later bit is great. This bit is nicer as a list of ~features~ rather than specific changes 👍🏻